src/it/projects/profile-with-deps-inherit-parent-depMgmt-flatten-dep-all-oss/bom/pom.xml

```diff
@@ -0,0 +1,58 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.codehaus.mojo.flatten.its</groupId>
+    <artifactId>profile-with-deps-inherit-parent-depMgmt-flatten-dep-all-oss-bom</artifactId>
+    <version>0.1-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <properties>
+        <javax.annotations.version>1.3.2</javax.annotations.version>
```

Comment:
shoulnd't the version be managed in the BOM?

Reply:
Already update